zapisy/apps/schedule/views.py

```diff
@@ -337,6 +337,50 @@ def get_queryset(self):
         return queryset
 
 
+def freedays(request):
+    """Returns list of freedays between 'start' and 'end' dates.
+
+    Both dates has to be given in 'input_date_format'. Date is returned in 'output_date_format'.
+    Returns 404 if one or both paramas are missing.
+    """
+    freedays = Freeday.objects.all()
+    try:
+        start = request.GET['start']
+        end = request.GET['end']
+    except Exception:
+        raise Http404
+    input_date_format = "%Y-%m-%dT%H:%M:%S.000Z"
+    output_date_format = "%Y-%m-%d"
+    start = datetime.datetime.strptime(start, input_date_format)
+    end = datetime.datetime.strptime(end, input_date_format)
+    formatted_start = start.strftime(output_date_format)
+    formatted_end = end.strftime(output_date_format)
+    response = freedays.filter(day__gte=formatted_start, day__lte=formatted_end).values()
```

Comment:
Django i tak zapewne dopiero tutaj wykona zapytanie do bazy, ale gdyby zrobić
```suggestion
    response = Freeday.objects.filter(day__gte=formatted_start, day__lte=formatted_end).values()
```
i oczywiście wyrzucić wcześniejsze odwołanie do `Freeday.objects`, to stałoby się to jawne.